spec/lib/dispatch-rider/subscriber_spec.rb

```diff
@@ -3,25 +3,25 @@
 describe DispatchRider::Subscriber do
 
   before do
-    DispatchRider::Handlers::Base.stub(:subclasses){ Set.new }
+    allow(DispatchRider::Handlers::Base).to receive(:subclasses) { Set.new }
 
     stub_const("FooBar", Class.new(DispatchRider::Handlers::Base) {
-      def process(options)
-        throw :process_was_called
-      end
-    })
+                         def process(options)
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `process(*)` if you want the method to accept any arguments but don't care about them.